crs_application/src/task_managers/scan_acquisition_manager.cpp

```diff
@@ -71,20 +95,69 @@ common::ActionResult ScanAcquisitionManager::verify()
 
 common::ActionResult ScanAcquisitionManager::moveRobot()
 {
-  RCLCPP_WARN(node_->get_logger(),"%s not implemented yet",__PRETTY_FUNCTION__);
-  return true;
+  auto freespace_motion_request = std::make_shared<crs_msgs::srv::CallFreespaceMotion::Request>();
+  freespace_motion_request->target_link = framos_frame_id_;
+  freespace_motion_request->goal_pose = scan_positions_.at(scan_index_);
+  freespace_motion_request->execute = true;
+
+  auto result_future = call_freespace_motion_client_->async_send_request(freespace_motion_request);
+  if (rclcpp::spin_until_future_complete(node_, result_future) !=
+    rclcpp::executor::FutureReturnCode::SUCCESS)
+  {
+    RCLCPP_ERROR(node_->get_logger(), "Call Freespace Motion service call failed");
+    // todo(ayoungs): do I need to reset the scan_index_ here and clear the point clouds?
```

Comment:
If motion planning or execution fails we could stop acquisition and store the point clouds that were gathered thus far as long as there's at least one, implementing this logic might require a tweak in the SM